crates/noirc_evaluator/src/brillig/brillig_gen.rs

```diff
@@ -48,7 +52,7 @@ impl BrilligGen {
     /// Converts an SSA Basic block into a sequence of Brillig opcodes
     fn convert_block(&mut self, block_id: BasicBlockId, dfg: &DataFlowGraph) {
         self.obj.add_block_label(block_id);
-        let block = &dfg[dbg!(block_id)];
```

Comment:
what did dbg! used to do? (generally not a huge fan of these acronym names, unless comments follow them everywhere :D)

Reply:
Doh. That would be more obvious if not next to dfg